xtask/Cargo.toml

```diff
@@ -7,3 +7,5 @@ edition = "2018"
 
 [dependencies]
 clap = { version = "*", features = ["derive"] }
+serde = { version = "1.0", features = ["derive"] }
+toml = "*"
```

Comment:
Should we be using exact versions here?  I know we didn't for clap, but we probably should fix the version for that too.

Reply:
i am used to, but i think it should work without

Reply:
Please use specific versions.